raft/rafttest/network.go

```diff
@@ -7,17 +7,25 @@ import (
 	"github.com/coreos/etcd/raft/raftpb"
 )
 
-type network interface {
+// a network interface
+type iface interface {
 	send(m raftpb.Message)
 	recv() chan raftpb.Message
+	disconnect()
+	connect()
+}
+
+// a network
+type network interface {
 	// drop message at given rate (1.0 drops all messages)
 	drop(from, to uint64, rate float64)
 	// delay message for (0, d] randomly at given rate (1.0 delay all messages)
 	// do we need rate here?
 	delay(from, to uint64, d time.Duration, rate float64)
-
 	disconnect(id uint64)
 	connect(id uint64)
```

Comment:
an network interface has at least two ends. one is on your local computer, another one is on the end router.